src/api/app/controllers/webui/user_controller.rb

```diff
@@ -59,6 +59,10 @@ def show
     return unless User.current == @displayed_user
     @is_displayed_user = User.current == @displayed_user
     @patchinfos = @displayed_user.involved_patchinfos
+    @password_changable = @configuration['change_password'] && CONFIG['proxy_auth_mode'] != :on && CONFIG['ldap_mode'] != :on
```

Comment:
What is the difference between @configuration and CONFIG?
And maybe it is worth to introduce a function in the configuration model where you check that?